mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -248,12 +269,38 @@ class GBTClassificationModel private[ml](
     if (prediction > 0.0) 1.0 else 0.0
   }
 
+  override protected def predictRaw(features: Vector): Vector = {
```

Comment:
sorry I'm a bit confused, this classifier also only deals with two classes, it does not support multiclass data.  Instead of overriding, what is the alternative?  There is no default predictRaw or raw2probability implemented in probabilistic classifier, and it seems that this is the minimum required for GBTClassifier to use ProbabilisticClassifier.  Can you please give more information on this point?